platform/admin/core/admin-core-migration-commands/src/main/java/org/codice/ddf/migration/commands/ExportCommand.java

```diff
@@ -50,14 +46,17 @@
 
     private final ConfigurationMigrationService configurationMigrationService;
 
+    private final Security security;
+
     private final Path defaultExportDirectory;
 
     @Argument(index = 0, name = "exportDirectory", description = "Path to directory to store export", required = false, multiValued = false)
     String exportDirectoryArgument;
 
     public ExportCommand(ConfigurationMigrationService configurationMigrationService,
```

Comment:
Might be worth noting for later -- but when I look at the class `ConfigurationMigrationService`, I infer that it will only iterate the implementors of `ConfigurationMigratable` and ignore `DataMigratable`. Perhaps it's worth a refactor to just `MigrationService` later on?